google/cloud/aiplatform/jobs.py

```diff
@@ -1564,13 +1653,45 @@ def network(self) -> Optional[str]:
         self._assert_gca_resource_is_available()
         return getattr(self._gca_resource.trial_job_spec, "network")
 
+    def _get_web_access_uris(self):
+        """Helper method to get the web access uris of the hyperparameter job"""
+        if self.trials:
+            self._web_access_uris = [
```

Comment:
Perhaps this should be a nested dictionary of trial id -> worker id -> web_access_uris based on the usage below as it captures that trial ids and workers ids are unique.